addons/apecloud-mysql/config/mysql-scale-vtgate-config-effect-scope.yaml

```diff
@@ -29,4 +29,6 @@ dynamicParameters:
   - read_write_splitting_ratio
   - read_after_write_consistency
   - read_after_write_timeout
-  - ddl_strategy
\ No newline at end of file
+  - ddl_strategy
+  - enable_display_sql_execution_vttablets
+  - enable_read_write_split_for_read_only_txn
```

Comment:
Dynamic parameters require WeScale's capability to hot-reload them.
Therefore, you need to release a new version of WeScale, and update the cluster version to point to this new release.